<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>519</width>
     <height>306</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>氨气浓度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_ammonia_concentration"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>甲苯浓度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_toluene_concentration"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>丙醇浓度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_propanol_concentration"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_start_collect">
        <property name="text">
         <string>开始采集</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>文件名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_file_input"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_choose_file">
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="te_result"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
